bigtable/spark/README.md

```diff
@@ -0,0 +1,85 @@
+# Spark Applications for Cloud Bigtable
+
+## Overview
+
+The project shows how to load or save data to [Cloud Bigtable](https://cloud.google.com/bigtable) using [Apache Spark](https://spark.apache.org/) and [Apache HBase™ Spark Connector](https://github.com/apache/hbase-connectors/tree/master/spark) . 
+
+The project uses [sbt](https://www.scala-sbt.org/) as the build tool.
+
+## Prerequisites
+
+1. A [Google Cloud project](https://console.cloud.google.com/) with billing enabled.
+Please be aware of [Cloud Bigtable](https://cloud.google.com/bigtable/pricing) pricing.
+
+1. [Google Cloud SDK](https://cloud.google.com/sdk/) installed.
+
+1. [sbt](https://www.scala-sbt.org/) installed.
+
+1. [Apache Spark](https://spark.apache.org/) installed. Download Spark built for Scala 2.11.
+
+1. A basic familiarity with [Apache Spark](https://spark.apache.org/) and [Scala](https://www.scala-lang.org/).
+
+## Assemble the Example
+
+The Spark application can be assembled into an uber/fat jar with all of its dependencies and configuration. To build use `sbt` as follows:
+
+```
+sbt clean assembly
+```
+
+The above command should build `target/scala-2.11/bigtable-spark-samples-assembly-0.1.jar` file.
+
+**NOTE**: Since the Cloud Bigtable configuration is included in the (fat) jar, any changes will require re-assembling it.
+
+## Start Bigtable Emulator
```

Comment:
All of these steps should be under a single section "Test using the Bigtable emulator"

Something like (note I'm using 4 backticks so that this comment would render)
```
## Test using the Bigtable emulator
``
# Build an uber/fat jar
sbt clean assembly

# Start the emulator
gcloud beta emulators bigtable start
$(gcloud beta emulators bigtable env-init)

# Start the job
$SPARK_HOME/bin/spark-submit \
  --packages org.apache.hbase.connectors.spark:hbase-spark:1.0.0 \
  target/scala-2.11/bigtable-spark-samples-assembly-0.1.jar \
  $BIGTABLE_TABLE $NUMBER_OF_ROWS

# Verify 
$ cbt \
  -project=your-google-cloud-project \
  -instance=your-bigtable-instance \
  ls
wordcount
``

```

Then we can add a section in the future for dataproc and real bigtable

Reply:
considered done